### PR TITLE
complete dark mode

### PR DESCRIPTION
**Pull request by Scot-MacDonald**

### CODE REVIEW

File: JS-basics/js-basics_dark-mode/js/index.js [hunk 1]
```diff
@@ -1,3 +1,21 @@
 console.clear();
 
 const bodyElement = document.querySelector('[data-js="body"]');
+const dark = document.querySelector('[data-js="dark-mode-button"]');
+const light = document.querySelector('[data-js="light-mode-button"]');
+const toggle = document.querySelector('[data-js="toggle-button"]');
+
+dark.addEventListener("click", () => {
+  bodyElement.classList.add("button--light");
```
Comment (kayachakmak) [inline JS-basics/js-basics_dark-mode/js/index.js]: line 9 actually adds light mode when you click on the button? Why? we only want to add dark mode. you dont need that code

File: JS-basics/js-basics_dark-mode/js/index.js [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+  bodyElement.classList.add("button--dark");
+});
+
+light.addEventListener("click", () => {
+  bodyElement.classList.remove("button--dark");
+  bodyElement.classList.add("button--light");
```
Comment (kayachakmak) [inline JS-basics/js-basics_dark-mode/js/index.js]: light mode button only removes dark features from the html code. you dont need ot add light  mode, default is already light.

File: JS-basics/js-basics_dark-mode/js/index.js [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+});
+
+toggle.addEventListener("click", () => {
+  bodyElement.classList.toggle("button--dark");
```
Comment (kayachakmak) [inline JS-basics/js-basics_dark-mode/js/index.js:19]: toggle button only toggles the dark feature. you dont need to add code regarding light features.

**Review by kayachakmak — COMMENTED** [review]
Your code needs some changes, I explained in the comments. 

Best luck!

File: JS-basics/js-basics_dark-mode/js/index.js (hunk 3 above)
Comment (Scot-MacDonald) [reply JS-basics/js-basics_dark-mode/js/index.js:19]: Done all the changes you recommended Kaya. Thanks for your review - top!

File: JS-basics/js-basics_dark-mode/js/index.js [hunk 4]
```diff
@@ -6,16 +6,16 @@ const light = document.querySelector('[data-js="light-mode-button"]');
 const toggle = document.querySelector('[data-js="toggle-button"]');
 
 dark.addEventListener("click", () => {
-  bodyElement.classList.add("button--light");
+  //   bodyElement.classList.add("button--light");
```
Comment (kayachakmak) [inline JS-basics/js-basics_dark-mode/js/index.js:9]: Thanks for the updates Scot! However I think the code still doesnt work. Have you tested it?

"bodyElement.classList.add("button--light");

 the  classList.add function adds a class that is indicated in paranthesis ("button-light") to the element (bodyElement)

the problem is here, button--light is a class for the button, not for the element you want to see the change. 

If you check the CSS file, you can see there is a class for the dark mode. (body.dark) 

If you write your code as bodyElement.classList.add("dark"), this code adds dark class to your body element. (body.dark)

I hope I was able to explain. :)

**Pull request merged**